helpdesk-bot/src/test/resources/stories/alternativeBotScenarios.story

```diff
@@ -21,7 +21,7 @@ Then bot can verify the agent1 user was added to the ticket room
 And agent1 user can see all the personal history conversation in the ticket room
 And helpdesk_client1 can verify the ticket claimed message in the client room
 When agent1 user sends a message to close the ticket
-Then bot can verify there are no agents in the ticket room
+Then bot can verify there are no agent in the ticket room
```

Comment:
"Then bot can verify there are no agents in the ticket room".

This comment should be applied to all scenarios.